docs/extras/xdebug.md

```diff
@@ -14,6 +14,11 @@ To use XDebug with PHPStorm, change the `php_xdebug_idekey` variable as shown be
 ```yaml
 php_xdebug_idekey: PHPSTORM
 ```
+In order to pass the Web Server Debug Validation, you'll need to also add the following your `config.yml`
+
+```yaml
+php_xdebug_remote_host: "{{ vagrant_hostname }}"
```

Comment:
`php_xdebug_remote_host` should be set to the IP/hostname of the host machine.

`vagrant_hostname` is the hostname given to the VM.

With default settings the host IP this is either `10.0.2.2` or `192.168.88.1` but it could be different depending on your network configuration.